<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteFrame</class>
 <widget class="QFrame" name="NoteFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>302</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="toLabel">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="toEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="subjectLabel">
       <property name="text">
        <string>Subject</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="subjectEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="bodyTextEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="signatureBox">
     <property name="text">
      <string>Include signature</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="draftButton">
       <property name="text">
        <string>Save as draft</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="previewButton">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
